README.md

```diff
@@ -82,27 +88,11 @@ Start Elasticsearch from the `elasticsearch` directory:
         $ cd elasticsearch/
         $ ./dev-start.sh
 
-### Create CloudFormation Stack
-
-First you need to create some dev credentials in AWS - ask your friendly system administrator.
-
-**Pro-tip**: Set `AWS_DEFAULT_PROFILE` to avoid using the `--profile` flag with the awscli in the future.
-
-```sh
-echo 'export AWS_DEFAULT_PROFILE=media-service' >> $HOME/.profile
-```
-
-To create your stack run [create-dev-stack.sh](cloud-formation/scripts/create-dev-stack.sh):
-
-```sh
-cd cloud-formation/scripts
-./create-dev-stack.sh
-```
-
-** The create-dev-stack script will use your default aws profile **
 
 ### Generate .properties files
 
+Firstly you will need to follow the guide in the dev-utils folder of this [repo](https://github.com/guardian/grid-infra). This will give you 
```

Comment:
indeed, which is why we figured it was safe to link to it, however, perhaps we should actually be more explicit about that in the README? (e.g. "Guardian devs can use this private repo to get help with their local setup").